RELEASE/scripts/autoscend/auto_familiar.ash

```diff
@@ -682,6 +682,17 @@ void preAdvUpdateFamiliar(location place)
 		}
 	}
 
+	if (my_familiar() == $familiar[Reagnimated Gnome])
+	{
+		// This arena visit to obtain gnome familiar equips is turn free and can be done once a day.
+		if (!get_property("_auto_gnomeArenaVisited").to_boolean())
+		{
+			visit_url("arena.php");
+			set_property("_auto_gnomeArenaVisited", "true");
```

Comment:
```suggestion
			visit_url("arena.php");
			run_choice();
			set_property("_auto_gnomeArenaVisited", "true");
```
Otherwise mafia will get stuck in the arena choice.
Sorry missed this out in original review.

Reply:
Hmm, this change broke things. run_choice needs an int supplied. My (probably incorrect) perception was that auto_run_choice would take over if a surprise choice adventure sprung up.

Reply:
The problem with using visit_url is that you're effectively circumventing all of mafia's handling so you need to handle everything from that point until whatever you're doing is done.

I made a mistake in the suggested code. It should've been passing -1 to run_choice() so it uses your choice properties or choiceAdventureScript.